docs/sources/operators-guide/running-production-environment/production-tips/index.md

```diff
@@ -30,6 +30,13 @@ The required disk space depends on the number of time series stored in the inges
 
 For more information about estimating the required ingester disk space, refer to [Planning capacity]({{< relref "../planning-capacity.md#ingester" >}}).
 
+### Ingester disk IOPS
+
+The ingester disk performances (IOPS and latency) can impact both write and read requests.
+On this write path, ingester disk is touched to write the write-ahead log (WAL), while on the read path the ingester disk is touched to query series whose chunks have already been written to disk.
+
+We recommend to run ingesters with a fast disk (e.g. SSD disk).
```

Comment:
```suggestion
On the write path, the ingester disk is touched (via the `touch` command) to write the write-ahead log (WAL), and on the read path the ingester disk is touched to query series whose chunks have already been written to disk.

For that reason, use ingesters with fast disk speed such as SSDs.
```
In plain language, "touched" seems to refer to the `touch` command. Is this accurate or do you mean something else?

Do you intend "while" to be simultaneous or do you mean "and"?

What does "is touched to query series" mean?

Reply:
Touched here is not related to the "touch" command. What I meant with touch is "either read or write from disk".

Reply:
Looks like "touch" can mean two things here. How is this latest suggestion wrt accuracy?

Reply:
> > On the write path, the ingester disk writes to the write-ahead log (WAL).
> 
> It's not the "ingester disk" that writes, but it's the ingester that writes the WAL to disk.
> 
> _Same comment applies to suggestion read on read path._

I have updated my suggestion.